### ISSUE TITLE
importance of 0

### ISSUE THREAD

**Issue by lhambrid** [imbs-hl/ranger#636]
Hello, I'm having a problem with ranger sometimes reporting importances of 0 for all predictors.
For example, this:
`rf <- ranger(RATING ~ NFCr + AHCr + IHCr + Mediation + InanCEAF + ContrHCEAF + PhysImpHCEAF + PsychImpHCEAF + LangSpecRT, data=df2, importance = "permutation", num.trees = num_trees, mtry = num_vars_sampled_V1_V3)

print(importances(rf))`

sometimes produces this, depending on what the input is:
NFCr  AHCr  IHCr  Mediation  InanCEAF
  0        0         0        0               0
ContrHCEAF  PhysImpHCEAF  PsychImpHCEAF
  0                     0                          0

I have a strong hunch that this is an error, but my data is clean so I really don't know where it might be coming from.  Any ideas?

**Comment by mnwright** [comment]
This happens when the OOB error is unchanged after permuting a predictor. Maybe the predictors are highly correlated? In any case, it's hard to say without a reproducible example.

**Comment by lhambrid** [comment]
The dependent variable and all of the predictors are binary in nature, with values of either "yes" or "no."  I looked at it more closely, and it appears that the OOB prediction error is very low in each of these 0 importance cases, no greater than 12% and usually much lower, as low as 0.5%.  I noticed that in these cases, 90% or more (in fact, exactly the inverse of the OOB error) of the correct values of "Rating" are "yes".  Is it possible that the model is bypassing the predictors altogether, just guessing "yes" every time, and therefore succeeding 90% of the time?  Can it choose to not even try using the predictors?  For 600 trees??
